src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -68,6 +95,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _contentAutomationPeer.SetControlPadding(padding);
     }
 
+    void TermControlAutomationPeer::RecordKeyEvent(const WORD vkey)
+    {
+        if (const auto charCode{ MapVirtualKey(vkey, MAPVK_VK_TO_CHAR) })

```

Comment:
something tells me there will be future bugs here related to unicode input, but this is probably fine for now. 

My memory harkens back to `Terminal::_CharacterFromKeyEvent`

Reply:
oh but there's the `towupper` thing below... That might be more complicated than the `Terminal` version. Still, probably good enough for now.